ChocolArm64/Translator.cs

```diff
@@ -0,0 +1,165 @@
+using ChocolArm64.Decoders;
+using ChocolArm64.Events;
+using ChocolArm64.Memory;
+using ChocolArm64.State;
+using ChocolArm64.Translation;
+using System;
+using System.Reflection.Emit;
+
+namespace ChocolArm64
+{
+    public class Translator
```

Comment:
Was there a reason everything got renamed from `Axxx` to `Cpuxxxx` except for `ATranslator`? If not, would it not make sense for this to be named `CpuTranslator` too?

```suggestion
    public class CpuTranslator
```

Reply:
I'm fine with calling it Translator, but if it is renamed to CpuTranslator, then I guess that TranslatorCache would also need to be renamed.